README.md

```diff
@@ -17,6 +17,7 @@ It is a standalone Microservice that connects the two cloud platforms and provid
 
 - [Documentation](#documentation)
 - [For developers and contributors](#for-developers-and-contributors)
+- [For shop integration (front/back end developers)](#for-shop-integration-frontback-end-developers)
```

Comment:
Why is it not part of "Using the Integration in a project"?

Reply:
merged two points to one: 54b58e3